solr/packaging/test/bats_helper.bash

```diff
@@ -20,7 +20,7 @@
 #   The SOLR_HOME directory will be cleared when the next test file is executed.
 # - "setup" should use "common_setup" if a Solr process is NOT being started in that same "setup" function.
 common_setup() {
-    bats_require_minimum_version 1.8.0
+    bats_require_minimum_version 1.8.2
```

Comment:
Is this line below still needed?

`load "${BATS_LIB_PREFIX}/bats-support/load.bash"`

Reply:
Not sure. `bats-support` is a npm dependency of `bats-assert`, so it is still being used, perhaps it is auto loaded somewhere?